src/less/tokens/base/tokens.less

```diff
@@ -70,6 +70,7 @@
 // status
 @color-status-attention: @color-r4;
 @color-status-information: @color-b4;
+@confirmation-icon-color: #28a443;
```

Comment:
We dont have a variable for this? I think it should be there.

Reply:
I was presuming this was part of the temporary hardcoding to avoid any collisions/movement of `@color-g4`. Plus I've been having a really hard time with vars getting overridden and wrong ones being inherited. I can try it again.

Reply:
Yes, to help reduce merge conflicts, let's move this out of tokens.less and into icon.less. You can use `@color-g4`.

https://github.com/eBay/skin/blob/da34f53cd377ec758ab3eb417aec69d33da0a495/src/less/primitives/ds6/primitives.less#L58

Reply:
So I moved the var into `icon.less`:

```
@import "../../mixins/utility/utility-mixins.less";
@confirmation-icon-color: #28a443; // < THIS WORKS
@confirmation-icon-color: @color-g4; // < THIS DOESN'T WORK
```

This is what I was referring to as things are getting overridden.

<img width="784" alt="image" src="https://user-images.githubusercontent.com/1675667/165145178-d4835625-6030-422e-aabd-e2cac9632544.png">

Notice how it reverted back to `#5ba71b`.

Reply:
That looks like the DS4 color. Is that being pulled in somehow?

Reply:
I think something funny is going on.

In the dist files I see the correct color but on the docs it seems off. 
I think try doing a clean should work.